src/components/MultiHopTrade/components/SpotTradeSuccess/SpotTradeSuccess.tsx

```diff
@@ -325,22 +179,14 @@ export const SpotTradeSuccess = ({
               <Text translation={titleTranslation} fontWeight='bold' />
             </Stack>
             <AmountsLine />
-            <SurplusLine />
-            {enableFoxDiscountSummary &&
-              hasFeeSaving &&
-              bnOrZero(feesOrTotalUpsideCryptoPrecision).gt(0) && (
-                <Box px={8}>
-                  <YouSaved
-                    totalUpsidePercentage={feesOrTotalUpsidePercentage}
-                    totalUpsideCryptoPrecision={feesOrTotalUpsideCryptoPrecision}
-                    sellAsset={sellAsset}
-                    buyAsset={buyAsset}
-                  />
-                </Box>
-              )}
-            {couldHaveReducedFee && affiliateFeeUserCurrency && (
+            {bnOrZero(maybeExtraDeltaCryptoPrecision).gt(0) && (
               <Box px={8}>
-                <YouCouldHaveSaved affiliateFeeUserCurrency={affiliateFeeUserCurrency} />
+                <YouGotMore
```

Comment:
This guy is still looking good!

<img width="522" alt="Screenshot 2025-05-14 at 5 31 20 pm" src="https://github.com/user-attachments/assets/60e93727-db16-4a78-ae76-a62166eb8d47" />